🌐 Add German translation for `docs/de/docs/tutorial/security/first-steps.md`

nilslindemann opened the pull request:
← `tutorial/security/index.md` (#10429)
→ `tutorial/security/get-current-user.md` (#10439)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
🌐 Add German translation for `docs/de/docs/tutorial/security/first-steps.md`

* Full translation
* In sync with current docs/en/docs/tutorial/security/first-steps.md
* Google Translate used
* Manually checked
* Checked for errors using LanguageTool

tiangolo commented:
📝 Docs preview for commit 6dc69ed6b82b6ea97d0c4c794a553ce297eb365a at: https://0be364c4.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/first-steps.md

tiangolo commented:
📝 Docs preview for commit f7f48885e09411a9122803ffe08d5fcc7463a1fd at: https://77de6397.fastapitiangolo.pages.dev

Commit:
Sync with #10442

tiangolo commented:
📝 Docs preview for commit 35d34f1f900b5ea1a08807d433623d2396ec6b1c at: https://eda3e2f1.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/first-steps.md

tiangolo commented:
📝 Docs preview for commit 099ab0962c7b9757f77e9b883cc41ee4c778c23a at: https://cca9660b.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/first-steps.md

Commit:
"Utility" -> "Werkzeug"

tiangolo commented:
📝 Docs preview for commit 8814bcb89ef669d62bf9319a8d295a8bf9da3dc5 at: https://f7d1a297.fastapitiangolo.pages.dev

Commit:
"Tip" -> "Tipp"

tiangolo commented:
📝 Docs preview for commit 5a19ae87f068ebaa792ef1894efad387e65e0778 at: https://34c54246.fastapitiangolo.pages.dev

Commit:
"Sicherheitssystem" -> "Sicherheitsschema"

tiangolo commented:
📝 Docs preview for commit 32cbc956784111d5261791c9aa0dfd6eacf97b4b at: https://68952dd7.fastapitiangolo.pages.dev

Commit:
"Das Token" -> "Der Token"

Sources:
https://www.korrekturen.de/genus/der-oder-das-token.shtml
https://rechtschreibung.woxikon.de/artikel/der-oder-das-token

Also fix a wrong translation

tiangolo commented:
📝 Docs preview for commit b32f0fce4a2034ebbcc7c9b539540fa12e716555 at: https://e3d49a93.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/first-steps.md

tiangolo commented:
📝 Docs preview for commit fc8794ed8dd3f8d92e093383ac957056b3b510af at: https://7e5de938.fastapitiangolo.pages.dev

Commit:
Use typographic quotes

tiangolo commented:
📝 Docs preview for commit 286d161c2dfd3a9402f588be4e05223e6274460e at: https://3b4259bd.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/first-steps.md

tiangolo commented:
📝 Docs preview for commit a0f784f1295b095c697a7d52b1e452d183e06da8 at: https://b17c195b.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/first-steps.md

tiangolo commented:
📝 Docs preview for commit f4f0bab869a59aa3f8577b69f50f1748f66c94ee at: https://0c1e9027.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/first-steps.md

tiangolo commented:
📝 Docs preview for commit b2aa8d2cd5b7962af025b3ed615bc0a98dd859f9 at: https://cf2452f1.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/first-steps.md

Commit:
"z.B." -> "z. B."

tiangolo commented:
📝 Docs preview for commit ca96e1d60b09f82e63dbfc545adbc63935fb00f0 at: https://a3f287af.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/first-steps.md

tiangolo commented:
📝 Docs preview for commit 1fef191022e9f3785df57243fa5dc0a46d79e453 at: https://cafa88ca.fastapitiangolo.pages.dev

tiangolo commented:
Awesome @nilslindemann! :rocket: 

As you are the champion of German translations and you have done so many, and with so much attention to detail, I'm gonna go ahead and merge this. :tada: 

If others see things that could be improved later, we can check that in new PRs. :nerd_face:
